src/ucp/rndv/proto_rndv.h

```diff
@@ -11,6 +11,8 @@
 
 #include <ucp/proto/proto_multi.h>
 
+#define UCP_PROTO_RNDV_ALIGN           256
```

Comment:
maybe instead use `iface_attr->cap.am.opt_zcopy_align` and change it to be 256? (now it is `UCS_SYS_PCI_MAX_PAYLOAD` which is 512)